release/gff/gff.sgw.gurage/source/gff.sgw.gurage.model.kps

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package>
+  <System>
+    <KeymanDeveloperVersion>13.0.100.0</KeymanDeveloperVersion>
+    <FileVersion>12.0</FileVersion>
+  </System>
+  <Options>
+    <ExecuteProgram></ExecuteProgram>
+    <ReadMeFile>readme.htm</ReadMeFile>
+    <MSIFileName></MSIFileName>
+    <MSIOptions></MSIOptions>
+  </Options>
+  <StartMenu>
+    <Folder></Folder>
+    <Items/>
+  </StartMenu>
+  <Info>
+    <Name URL="">Gurage</Name>
+    <Copyright URL="">© 2020 Ge'ez Frontier Foundation</Copyright>
+    <Author URL="">Ge'ez Frontier Foundation</Author>
+    <Version URL="">1.0</Version>
+  </Info>
+  <Files>
+    <File>
+      <Name>..\build\gff.sgw.gurage.model.js</Name>
+      <Description></Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.js</FileType>
+    </File>
+    <File>
+      <Name>welcome.htm</Name>
+      <Description></Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.htm</FileType>
+    </File>
+    <File>
+      <Name>readme.htm</Name>
+      <Description></Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.htm</FileType>
+    </File>
+  </Files>
+  <Keyboards/>
+  <LexicalModels>
+    <LexicalModel>
+      <Name>Gurage</Name>
+      <ID>gff.sgw.gurage</ID>
+      <Languages>
+        <Language ID="sgw">Sebat Bet Gurage</Language>

```

Comment:
Since the gff_gurage keyboard has the language ID `sgw-Ethi` , this should match

Reply:
I reviewed other GFF keyboards and to be consistent with them the "-Ethi" should be removed from the gff_gurage keyboard.  The gff_gurage keyboard does not have a tailored Touch Layout, I'll add one this weekend and make the identifier change.